src/services/utilServices/MailClient.ts

```diff
@@ -37,19 +40,26 @@ class MailClient {
   async sendMail(
     recipient: string,
     subject: string,
-    body: string
+    body: string,
+    attachments?: string[]
   ): Promise<void> {
     const sendEndpoint = `${POSTMAN_API_URL}/transactional/email/send`
-    const email = {
-      subject,
-      from: "IsomerCMS <donotreply@mail.postman.gov.sg>",
-      body,
-      recipient,
-      reply_to: "noreply@isomer.gov.sg",
-    }
+    const form = new FormData()
+    form.append("subject", subject)
+    form.append("from", "IsomerCMS <donotreply@isomer.gov.sg>")
+    form.append("body", body)
+    form.append("recipient", recipient)
+    form.append("reply_to", "noreply@isomer.gov.sg")
+    attachments?.forEach((attachment) => {
+      form.append(
+        "attachments",
+        fs.readFileSync(attachment),
+        attachment.split("/").pop()
```

Comment:
nit: use path.basename instead for clarity on what this line of code is trying to achieve